### PR TITLE
 The Big Emigration (ciao sdks)

### LINKED ISSUE #10800: remove sdks + cli from monorepo
only after https://github.com/celo-org/developer-tooling/issues/2 is complete

this the issue to remind is to come back and remove from one repo what will live in a new one.

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

Removes all of /packages/sdks,  packages/cli, etc  . These will now live at c[elo-org/developer-tooling](https://github.com/celo-org/developer-tooling/) see https://github.com/celo-org/developer-tooling/pull/12 for that diff 

This is not the end state of the transition, rather it is just the largest chunk. But its better to be swift rather than have big changes lingering for a long time

### Follow-up PRs will 

* upgrade yarn, linting, typescript etc, 
* remove more packages from root packge.json (once i figure out which are actually needed) 
* do more clean up for anything left 

### Other changes

fixes little issue in pre-push script. 
fixes little issue in dev-utils package.json

### noteworthy

* typescript package remains because as soon as we upgrade to eslint-tyescript we can just delete it. 
* dev-utils remains. I think we might move to a separate repo later. 

### Tested

existing tests

### Related issues

- Fixes #10800 
- part of https://github.com/celo-org/celo-labs/issues/1022

### Documentation

n/a

### CODE REVIEW

File: .github/workflows/celo-monorepo.yml [hunk 1]
```diff
@@ -170,45 +170,25 @@ jobs:
...
+
 
   protocol-test-release:
     name: Protocol Test Release
     runs-on: ['self-hosted', 'monorepo-node18']
     timeout-minutes: 500
     needs: [install-dependencies, lint-checks]
     if: |
-      github.base_ref == 'master' || contains(github.base_ref, 'staging') || contains(github.base_ref, 'production') ||
+      github.base_ref == 'master' || contains(github.base_ref, 'release') || contains(github.base_ref, 'production') ||
```
Comment (aaronmgdr) [inline .github/workflows/celo-monorepo.yml:190]: this is additional. we dont use git branches with staging in the branch. but we do with release

File: .github/workflows/celo-monorepo.yml [hunk 2]
```diff
@@ -71,13 +71,13 @@ jobs:
           restore-keys: |
             node-${{ runner.os }}-${{ runner.arch }}-${{ env.NODE_MODULE_CACHE_VERSION }}-
       - name: Install yarn dependencies
-        run: git config --global url."https://".insteadOf ssh:// && yarn install
+        run: git config --global url."https://".insteadOf ssh:// && yarn install --verbose --frozen-lockfile --network-timeout 1000000
```
Comment (aaronmgdr) [inline .github/workflows/celo-monorepo.yml]: added to debug will remove

File: package.json [hunk 3]
```diff
@@ -6,15 +6,13 @@
   "private": true,
   "scripts": {
     "preinstall": "git config --global url.\"https://\".insteadOf ssh://",
-    "install-pkg": "yarn install --link-duplicates",
     "lint": "yarn lerna run lint && yarn run --silent lint:do-not-merge",
     "lint:do-not-merge": "! git grep -E 'DO[ _]*NOT[ _]*MERGE'",
-    "prettify": "yarn run prettier --config .prettierrc.js --write '**/*.+(ts|tsx|js|jsx|sol|java)'",
-    "prettify:diff": "yarn run prettier --config .prettierrc.js --list-different '**/*.+(ts|tsx|js|jsx|sol|java)'",
+    "prettify": "yarn run prettier --config .prettierrc.js --write '**/*.+(ts|js|sol)'",
```
Comment (aaronmgdr) [inline package.json:11]: no java or j|tsx files exist in this repo

File: package.json [hunk 4]
```diff
@@ -6,15 +6,13 @@
...
+    "prettify:diff": "yarn run prettier --config .prettierrc.js --list-different '**/*.+(ts|js|sol)'",
     "reset": "yarn reset-modules && yarn reset-cache",
-    "reset-cache": "yarn reset-yarn && yarn reset-rn",
+    "reset-cache": "yarn reset-yarn",
     "reset-modules": "rm -rf node_modules/ packages/*/node_modules",
-    "reset-rn": "watchman watch-del-all; rm -rf $TMPDIR/metro-cache-*; rm -rf $TMPDIR/haste-map-*; rm -rf $TMPDIR/metro-symbolicate*",
```
Comment (aaronmgdr) [inline package.json:17]: This command is related to reseting react-native and should have been remove long ago

File: package.json [hunk 5]
```diff
@@ -28,21 +26,12 @@
     "keys:decrypt": "bash scripts/key_placer.sh decrypt",
     "keys:encrypt": "bash scripts/key_placer.sh encrypt",
     "check:packages": "node ./scripts/check-packages.js",
-    "release": "yarn clean && yarn build && yarn --cwd packages/cli run prepack && yarn cs publish",
-    "beta-enter": "bash scripts/beta-mode.sh",
-    "beta-exit": "yarn changeset pre exit && git commit -am 'Complete beta mode'",
-    "version-and-reinstall": "yarn changeset version && yarn install",
```
Comment (aaronmgdr) [inline package.json:34]: These are related to changesets and it seems that will not be used on this repo

File: packages/env-tests/package.json [hunk 6]
```diff
@@ -10,7 +10,6 @@
     "@celo/utils": "^5.0.6",
     "@celo/base": "^6.0.0",
     "@celo/connect": "^5.1.1",
-    "@celo/phone-utils": "^6.0.0",
```
Comment (aaronmgdr) [inline packages/env-tests/package.json:13]: Was not used. Removed to fix an odd issue

File: packages/dev-utils/package.json [hunk 7]
```diff
@@ -12,9 +12,13 @@
   "keywords": [
     "celo"
   ],
+  "files": [
+    "lib/*"
+  ],
   "scripts": {
```
Comment (aaronmgdr) [inline packages/dev-utils/package.json:18]: Added repack and explicit files as libs because one published version of this had src files

**Comment by aaronmgdr** [comment]
most of the "failing" tests are those that this pr removes. the other is known to fail from time to time

**Review by shazarre — APPROVED** [review]

**Comment by aaronmgdr** [comment]
since the packages which run the tests are being remove in the pr we need to turn off the requirement that they pass to merge 

@lvpeschke Can you help here?

**Pull request merged**